src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -233,10 +244,14 @@ message LogMetricsConfig {
 message AdminConfig {
   string deployment_environment = 1;
 
+  string cluster_id = 4;
+
   oneof type {
     AwsAdminConfig aws_admin_config = 2;
     GcpAdminConfig gcp_admin_config = 3;
   }
+
+  bool enable_graceful_shutdown = 5;
```

Comment:
Is this flag in use here, I assume it would be used like `clusterId` is used in `Worker.java` above, right?